packages/metadata/src/rNFT/index.ts

```diff
@@ -0,0 +1,25 @@
+import { buildYup } from "schema-to-yup";
+import { SchemaOf } from "yup";
+import schema from "./schema.json";
+import { IMetadata } from "../iMetadata";
+
+export const rNFTMetadataSchema: SchemaOf<RNftMetadata> = buildYup(schema, {});
+
+// https://docs.opensea.io/docs/metadata-standards
+// https://github.com/ethereum/ercs/blob/master/ERCS/erc-721.md
+
+export type RNftMetadata = Omit<IMetadata, "type"> & {
+  type: "rNFT";
+  name: string;
+  description: string;
+  image?: string;
+  externalUrl: string;
+  licenseUrl: string;
+  condition?: string;
+  animationUrl?: string;
+  attributes?: {
+    trait_type: string;
```

Comment:
can we change it to?
```suggestion
    traitType: string;
```

Reply:
Done but had to keep compliance with NFT "standard" https://docs.opensea.io/docs/metadata-standards, at least for collection and nftItem